server/migrations/20190419001355-create-album.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Albums', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      artist: {
+        type: Sequelize.STRING
+      },
+      artistDescription: {
+        type: Sequelize.TEXT
+      },
+      albumTitle: {
+        type: Sequelize.STRING
+      },
+      tracks: {
+        type: Sequelize.TEXT
+      },
+      coverArt: {
+        type: Sequelize.STRING
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable('Albums');
+  }
+};
```

Comment:
This looks great! I've been trying to figure out my schema / seed scripts for postgres. I'm curious about what `up` and `down` are doing in `module.exports`? I haven't seen that before!

Reply:
the "up" method is the specified method that will be run by sequelize to seed the database while the "down" is the method that will be run by sequelize to drop the database.